Float.Core/Net/RequestClient.cs

```diff
@@ -307,7 +307,8 @@ async Task<Response> Send(HttpRequestMessage request)
 
                 // Verify if a redirect happened.
                 if (httpResponse.StatusCode == HttpStatusCode.Moved
-                    || httpResponse.StatusCode == HttpStatusCode.MovedPermanently)
+                    || httpResponse.StatusCode == HttpStatusCode.MovedPermanently
```

Comment:
According to [the docs](https://learn.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=net-9.0), `MovedPermanently` is a synonym for `Moved`.
```suggestion
```